packages/plugin-doc-coverage/src/lib/runner/doc-processer.integration.test.ts

```diff
@@ -0,0 +1,23 @@
+import { processDocCoverage } from './doc-processer.js';
+
+describe('processDocCoverage', () => {
+  it('should succesfully get the right number of ts files', () => {
+    const results = processDocCoverage({
+      sourceGlob: [
+        'packages/plugin-doc-coverage/mocks/fixtures/angular/**/*.ts',
+      ],
+    });
+    expect(results).toMatchSnapshot();
+  });
+
+  it('should succesfully get the right number of ts files and not include spec files', () => {
+    const results = processDocCoverage({
+      sourceGlob: [
+        'packages/plugin-doc-coverage/mocks/fixtures/angular/**/*.ts',
+        '!**/*.spec.ts',
+        '!**/*.test.ts',
+      ],
+    });
+    expect(results).toMatchSnapshot();
+  });
```

Comment:
I would use no snapshot here and maybe make the description more intuitive.

"should succesfully get the right number of ts files" => right number of files? what does it mean?

Reply:
Like from the source glob, it matches how many files would had match... but looking at it again, this tests looks pretty silly, I am gonna redo them